packages/currency/src/token.ts

```diff
@@ -0,0 +1,56 @@
+import { RequestLogicTypes } from '@requestnetwork/types';
+import { Currency } from './currency';
+import { getSupportedERC20Tokens } from './erc20';
+
+/**
+ * @class Token extends Currency to reference the symbol
+ */
+export class Token extends Currency {
+  constructor(
+    public value: string,
+    public type: RequestLogicTypes.CURRENCY,
+    public symbol: string,
+    public network?: string,
+  ) {
+    super({ value, type, network });
+  }
+
+  /**
+   * Gets a supported currency from a symbol, symbol-network or address.
+   * Iterates over all the supported networks if needed
+   * @param symbolOrAddress e.g. 'DAI', 'FAU', 'FAU-rinkeby' or '0xFab46E002BbF0b4509813474841E0716E6730136'
+   * @returns an ICurrency object
```

Comment:
copy paste? I don't correct it because i don't see the point of these `@returns` comments...